common/core/src/main/java/com/vmware/connectors/common/utils/HashUtil.java

```diff
@@ -0,0 +1,20 @@
+package com.vmware.connectors.common.utils;
+
+import org.apache.commons.codec.digest.DigestUtils;
+
+public final class HashUtil {
+
+    public static String hash(String... args) {
```

Comment:
Make this `Object... args` so callers aren't forced to turn the args into strings.